specification/compute/resource-manager/Microsoft.Compute/preview/2016-04-30-preview/compute.json

```diff
@@ -5730,6 +5828,58 @@
         }
       },
       "description": "Operation status response"
+    },
+    "InGuestSoftwareItemsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/InGuestSoftwareItem"
+          },
+          "description": "The list of InGuest software items on the VM."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The uri to fetch the next page of InGuest software items on the VM. Call ListNext() with this to fetch the next page of items."
+        }
+      },
+      "description": "The List of Software items operation response."
+    },
+    "InGuestSoftwareItemProperties": {
+      "properties": {
+        "name": {
```

Comment:
top level property names should not be repeated inside the properties bag. Lets rename this to softwareName